pkg/crcontrollers/drive/drivecontroller.go

```diff
@@ -215,6 +215,22 @@ func (c *Controller) handleDriveUpdate(ctx context.Context, log *logrus.Entry, d
 		return c.handleDriveUsageRemoved(ctx, log, drive)
 	case apiV1.DriveUsageFailed:
 		if c.checkAndPlaceStatusInUse(drive) {
+			volumes, err := c.crHelper.GetVolumesByLocation(ctx, id)
+			if err != nil {
+				return ignore, err
+			}
+			for _, vol := range volumes {
+				value, found := vol.GetAnnotations()[apiV1.VolumeAnnotationRelease]
+				if found && value == apiV1.VolumeAnnotationReleaseFailed {
+					vol.Spec.Usage = apiV1.VolumeUsageInUse
+					delete(vol.Annotations, apiV1.VolumeAnnotationRelease)
+					if err := c.client.UpdateCR(ctx, vol); err != nil {
+						log.Errorf("Unable to change volume %s usage status to %s, error: %v.",
+							vol.Name, vol.Spec.Usage, err)
+						return ignore, err
```

Comment:
Will we retry if update failed (by timeout, for example)?

Reply:
Hmm, if it failed, we [will return](https://github.com/dell/csi-baremetal/blob/d9a2e40a36b06d5857794139e4596f843afc227d/pkg/crcontrollers/drive/drivecontroller.go#L121) an error and don't update anything(disk/volume), then the parent's method(which called `Reconcile`) will get an unsatisfactory response and it makes more sense that **it** will try to send an update request again and again until the error disappears or the time runs out.